src/components/Tooltip/Tooltip.tsx

```diff
@@ -18,7 +19,75 @@ const fadeAnimation = keyframes`
     }
 `;
 
-const TooltipBody = styled.div<Pick<TooltipProps, 'inverted'>>`
+const arrowPlacementStyles = variant({
+    variants: {
+        bottom: {
+            right: 'calc(50% - 0.25rem)'
+        },
+        'bottom-end': {
+            right: '0.3rem'
+        },
+        'top-start': {
+            bottom: '-0.5rem',
+            transform: 'rotate(-180deg)'
+        },
+        top: {
+            bottom: '-0.5rem',
+            transform: 'rotate(-180deg)',
+            right: 'calc(50% - 0.25rem)'
+        },
+        'top-end': {
+            bottom: '-0.5rem',
+            transform: 'rotate(-180deg)',
+            right: '0.3rem'
+        },
+        left: {
+            top: 'calc(50% - 0.25rem)',
+            left: 'auto',
+            right: '-0.5rem',
+            transform: 'rotate(90deg)'
+        },
+        'left-end': {
+            bottom: '0.5rem',
+            left: 'auto',
+            right: '-0.5rem',
+            transform: 'rotate(90deg)'
+        },
+        'left-start': {
+            top: '0.5rem',
+            left: 'auto',
+            right: '-0.5rem',
+            transform: 'rotate(90deg)'
+        },
+        right: {
+            top: 'calc(50% - 0.25rem)',
+            left: '-0.25rem',
+            right: 'auto',
+            transform: 'rotate(-90deg)'
+        },
+        // TODO: Fix arrow placement
+        'right-end': {
+            bottom: '0.5rem',
+            left: '-0.25rem',
+            right: 'auto',
+            transform: 'rotate(-90deg)'
+        },
+        // TODO: Fix arrow placement
```

Comment:
What's the issue here? I don't think we can publish a component with broken arrows

Reply:
wow, nelson strikes again. forgot to remove the comments.